.changeset/funny-beans-vanish.md

```diff
@@ -0,0 +1,5 @@
+---
+"@biomejs/biome": patch
+---
+
+Fix a CLI flag `--bracket-spacing` is duplicated between the global configuration and the lang-specific override for JavaScript.
```

Comment:
```suggestion
Fixed the flag `--bracket-spacing` that was duplicated between the global configuration and the language-specific override for JavaScript.
```

Reply:
@arendjr Thank you for helping my bad English 😓